WalletWasabi.Gui/Controls/WalletExplorer/SendControlView.xaml

```diff
@@ -35,6 +35,8 @@
             </i:Interaction.Behaviors>
           </controls:ExtendedTextBox>
 
+          <controls:ExtendedTextBox IsVisible="{Binding IsCustomChangeAddress}" Classes="monospaceFont" Text="{Binding CustomChangeAddress}" Watermark="Custom Change Address" UseFloatingWatermark="True" />
```

Comment:
Missing PasteAddressOnClickBehavior. Is there a reason for this?

Reply:
Yes: https://github.com/zkSNACKs/WalletWasabi/pull/3198#issuecomment-595164113